pyrelational/data/data_manager.py

```diff
@@ -217,7 +217,10 @@ def set_target_value(self, idx: int, value: Any) -> None:
         :param idx: index value to the observation
         :param value: new value for the observation
         """
-        self.dataset.y[idx] = value
+        if hasattr(self.dataset, "y"):
```

Comment:
didn't know this was an issue :(, maybe we should have a attr_name input to data_manager that defaults to "y" or something? so users can adapt it

Reply:
Sure thing, I was retrofitting it here for some of the default datasets we have like the MNIST ones in the examples which uses the target attribute

Reply:
Ok, didn't see it was moved to a new issue!